test/apiv2/10-images.at

```diff
@@ -156,6 +156,17 @@ t GET images/json?filters='{"reference":["test1"]}' 200 length=1
 
 t DELETE libpod/images/test1:latest 200
 
+# to be used in prune until filter tests
+podman image build -t test1:latest -<<EOF
+from alpine
+RUN >file4
+EOF
+podman create --name test1 test1 echo hi
+
+t DELETE libpod/images/test1:latest 409
```

Comment:
The change is in the compat handler while the test is calling the libpod handler.
```suggestion
t DELETE images/test1:latest 409
```